module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,61 @@
+#include "compression.hpp"
+
+#include <iostream>
+
+void printMap(const bitmapArr& bitmap) {
+
+    for (const auto& arr : bitmap) {
+        for (auto el : arr) {
+            if (iscntrl(el))
+                std::cout << " ";
+            else
+                std::cout << el;
+        }
+        std::cout << "\n";
+    }
+}
+
+grayScaleVec compressGrayscale(const bitmapArr& grayScale) {
+
+    std::vector<std::pair<uint8_t, uint8_t>> compressed;
```

Comment:
You have declared alias for std::vector<std::pair<uint8_t, uint8_t>>. Why haven't you used it here?

Reply:
Sure, thanks

Reply:
I missed it